services/core-web/src/components/Forms/EditMinespaceUser.tsx

```diff
@@ -9,18 +8,22 @@ import { nullableStringSorter, resetForm } from "@common/utils/helpers";
 import RenderField from "@/components/common/RenderField";
 import * as FORM from "@/constants/forms";
 import { renderConfig } from "@/components/common/config";
-import CustomPropTypes from "@/customPropTypes";
+import { IMine } from "@mds/common";
```

Comment:
Could we import this from `@mds/common/interfaces` instead? Hoping to get rid of that root global export